src/BlockEntities/MobSpawnerEntity.cpp

```diff
@@ -136,18 +142,18 @@ void cMobSpawnerEntity::SpawnEntity(void)
 			auto & Random = GetRandomProvider();
 
 			bool HaveSpawnedEntity = false;
-			for (size_t i = 0; i < 4; i++)
+			for (size_t i = 0; i < m_SpawnCount; i++)
 			{
-				if (NearbyEntities >= 6)
+				if (NearbyEntities >= m_MaxNearbyEntities)
 				{
 					break;
 				}
 
 				auto SpawnRelPos(GetRelPos());
 				SpawnRelPos += Vector3i(
-					static_cast<int>((Random.RandReal<double>() - Random.RandReal<double>()) * 4.0),
+					static_cast<int>((Random.RandReal<double>() - Random.RandReal<double>()) * (float) m_SpawnRange),
 					Random.RandInt(-1, 1),
-					static_cast<int>((Random.RandReal<double>() - Random.RandReal<double>()) * 4.0)
+					static_cast<int>((Random.RandReal<double>() - Random.RandReal<double>()) * (float) m_SpawnRange)
```

Comment:
These two still have c-style casts, and m_SpawnRange should really be casted to double here.

Reply:
whats the c++ way to cast in that case`?

Reply:
```static_cast<double>(m_SpawnRange)```

Reply:
🤦 that obvious - my bad